libs/blocks/global-navigation/blocks/profile/profile.css

```diff
@@ -0,0 +1,133 @@
+:root {
+  --border-color: #e1e1e1;
+}
+
+.gnav-profile .gnav-profile-menu {
+  display: none;
+  position: fixed;
+  right: 0;
+  top: 56px;
+  min-width: 320px;
+  background: var(--color-white);
+  border: 1px solid var(--border-color);
+  border-radius: 0 0 4px 4px;
+  overflow: hidden;
+}
+
+.gnav-profile.is-open .gnav-profile-menu {
+  display: unset;
+}
+
+.gnav-profile-header {
+  display: flex;
+  padding: 20px;
+  align-items: start;
+}
+
+.gnav-profile-header .gnav-profile-img {
+  width: 64px;
+  height: 64px;
+  border-radius: 4px;
+}
+
+.gnav-profile-details {
+  margin-left: 16px;
+  align-self: stretch;
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.gnav-profile-name {
+  font-size: 18px;
+  line-height: 1;
+  margin: 0;
+  font-weight: 700;
+  flex: 0 0 100%;
+}
+
+.gnav-profile-email {
+  color: #707070;
+  font-size: 14px;
+  line-height: 1.25;
+  margin: 0;
+  flex: 0 0 100%;
+}
+
+.gnav-profile-account {
+  margin: 0;
+  text-decoration: underline;
+  font-size: 14px;
+  align-self: flex-end;
+  flex: 0 0 100%;
+}
+
+.gnav-profile-actions {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  font-size: 14px;
+}
+
+.gnav-profile-actions a {
+  padding: 14px 20px;
+  display: block;
+  border-top: 1px solid var(--border-color);
+  box-sizing: border-box;
+  color: #4b4b4b;
+}
+
+.gnav-profile-menu a:hover {
+  background-color: #FAFAFA;
+}
+
+header .gnav-profile.gnav-navitem.has-menu.is-open > a {
+  font-weight: unset;
+}
+
+.gnav-profile.gnav-navitem.has-menu.is-open .gnav-navitem-menu {
+  position: fixed;
```

Comment:
I don't understand the full picture, but `fixed` might not be the right approach here, as stated in a previous comment as well.

Reply:
Rewrote the thing